src/controllers/routes.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const requestUtil = require('five-bells-shared/utils/request')
+const routingTables = require('../services/routing-tables')
+const routeBroadcaster = require('../services/route-broadcaster')
+const knownConnectors = {}
+
+exports.post = function * () {
+  const routes = yield requestUtil.validateBody(this, 'Routes')
+
+  // TODO verify that POSTer of these routes matches route.connector.
```

Comment:
Hmm, we'll have to talk more about that. I'm not sure it makes sense to add routes to the incoming transfer. Even if we want to have connectors pay one another it might make more sense to pre-pay a bit and draw down against a balance rather than pay for every advertisement.